content/coronavirus_landing_page.yml

```
@@ -62,19 +62,18 @@ content:
   timeline:
     heading: Recent and upcoming changes
     list:
+    - heading: 27 October
+        paragraph: | 
```

The `p` of paragraph needs to line up with the `h` of heading:

```suggestion
    - heading: 27 October
      paragraph: | 
```